webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -1,117 +1,111 @@
 <template>
-  <ds-container width="small">
-    <ds-card v-if="success" class="success">
-      <ds-space>
-        <sweetalert-icon icon="success" />
-        <ds-text align="center" bold color="success">
-          {{ $t('registration.create-user-account.success') }}
-        </ds-text>
-      </ds-space>
-    </ds-card>
-    <ds-card v-else class="create-account-card">
-      <client-only>
-        <locale-switch />
-      </client-only>
-      <ds-space centered>
-        <img
-          class="create-account-image"
-          alt="Create an account for Human Connection"
-          src="/img/sign-up/nicetomeetyou.svg"
-        />
-      </ds-space>
-      <ds-space>
-        <ds-heading size="h3">
-          {{ $t('registration.create-user-account.title') }}
-        </ds-heading>
-      </ds-space>
+  <div v-if="response === 'success'">
+    <transition name="ds-transition-fade">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-09T14:21:07Z" title="Wednesday, October 9th 2019, 4:21:07 pm +02:00">Oct 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,117 +1,107 @@
 <template>
-  <ds-container width="small">
-    <ds-card v-if="success" class="success">
-      <ds-space>
-        <sweetalert-icon icon="success" />
-        <ds-text align="center" bold color="success">
-          {{ $t('registration.create-user-account.success') }}
-        </ds-text>
-      </ds-space>
-    </ds-card>
-    <ds-card v-else class="create-account-card">
-      <client-only>
-        <locale-switch />
-      </client-only>
-      <ds-space centered>
-        <img
-          class="create-account-image"
-          alt="Create an account for Human Connection"
-          src="/img/sign-up/nicetomeetyou.svg"
-        />
-      </ds-space>
-      <ds-space>
-        <ds-heading size="h3">
-          {{ $t('registration.create-user-account.title') }}
-        </ds-heading>
-      </ds-space>
+  <div v-if="response === 'success'">
+    <sweetalert-icon icon="success" />
```


it would be nice to wrap these in `<transition name="ds-transition-fade">` to keep it consistent across the code base